protocol/scripts/deploys/mainnet/v2/gnosis-strategy-1/01-gnosis-safe-circuit-breakers-dai.ts

```diff
@@ -1,10 +1,10 @@
 import '@nomiclabs/hardhat-ethers';
 import { ethers } from 'hardhat';
-import { TempleCircuitBreakerAllUsersPerPeriod__factory } from '../../../../typechain';
+import { TempleCircuitBreakerAllUsersPerPeriod__factory } from '../../../../../typechain';
 import {
   deployAndMine,
   ensureExpectedEnvvars,
-} from '../../helpers';
+} from '../../../helpers';
 
 async function main() {
```

Comment:
I don't think we have a valid gnosis strategy ready to deploy to mainnet yet. 

I think this can stay here as is -- when it comes time to deploy one for real we can just amend this one (and probably rename it/etc).

But we might want to comment out of the post-deploy script for now